e2e/process-services-cloud/process-custom-filters.e2e.ts

```diff
@@ -196,7 +488,58 @@ describe('Process list cloud', () => {
             expect(processCloudDemoPage.editProcessFilterCloudComponent().checkAppNamesAreUnique()).toBe(true);
             browser.actions().sendKeys(protractor.Key.ESCAPE).perform();
         });
-
     });
 
+    describe('Process List - Check Action Filters', () => {
+
+        beforeEach(async (done) => {
```

Comment:
For this describe i think it's not necessary to have so many filterProperties and sortingProperties, because are not used at all. I would keep only actions.

Reply:
agree